sklearn/decomposition/pca.py

```diff
@@ -120,6 +120,10 @@ class PCA(_BasePCA):
     Notice that this class does not support sparse input. See
     :class:`TruncatedSVD` for an alternative with sparse data.
 
+    Although Bessel correction decreases the bias of the estimator, it also
```

Comment:
This is still pretty confusing and apropos of nothing. I'm kinda tempted not to add the note if we can't come up with a good but not too-long explanation here.

"PCA uses the maximum likelihood estimate of the eigenvalues, which does not include the Bessel correction, though in practice this should rarely make a difference." ?